src/sentry/integrations/slack/webhooks/action.py

```diff
@@ -240,9 +240,20 @@ def validation_error(
             },
         )
 
-        text: str = list(*error.detail.values())[0]
+        text = self._unpack_error_text(error)
         return self.respond_ephemeral(text)
 
+    @staticmethod
+    def _unpack_error_text(validation_error: serializers.ValidationError) -> str:
+        detail = validation_error.detail
+        while True:
+            if isinstance(detail, dict):
+                detail = list(detail.values())
+            element = detail[0]
+            if isinstance(element, str):
+                return element
+            detail = element
```

Comment:
The `rest_framework` library defines the "detail" type as

```python
_Detail: TypeAlias = ErrorDetail | list[_Detail] | dict[str, _Detail]
```

Because of that recursively nested structure, this kind of loop is necessary to guarantee that we unpack a string (`ErrorDetail` extends `str`) without making assumptions about the structure as the original code (`list(*error.detail.values())[0]`) does. This satisfies type-checking and should be more robust.